GiEnJul/ClientApp/src/components/institution/FormFood.tsx

```diff
@@ -49,7 +49,7 @@ const FormDinner: FC<IFormDinner> = ({
   };
 
   useEffect(() => {
-    if (viewErrorTrigger) setViewError(true);
+    setViewError(!!viewErrorTrigger);
```

Comment:
why are you using double "!" here?

Reply:
This is to convert from int to boolean. E.g. converts first to false then true.

Reply:
aah thats JS for you